2021-05-28 03:56:13,490 DEBUG asyncio Using selector: EpollSelector
2021-05-28 03:56:13,495 DEBUG quic [3cf854ef2dee6084] TLS State.CLIENT_HANDSHAKE_START -> State.CLIENT_EXPECT_SERVER_HELLO
2021-05-28 03:56:13,707 DEBUG quic [3cf854ef2dee6084] QuicConnectionState.FIRSTFLIGHT -> QuicConnectionState.CONNECTED
2021-05-28 03:56:13,708 DEBUG quic [3cf854ef2dee6084] TLS State.CLIENT_EXPECT_SERVER_HELLO -> State.CLIENT_EXPECT_ENCRYPTED_EXTENSIONS
2021-05-28 03:56:13,821 DEBUG quic [3cf854ef2dee6084] TLS State.CLIENT_EXPECT_ENCRYPTED_EXTENSIONS -> State.CLIENT_EXPECT_CERTIFICATE_REQUEST_OR_CERTIFICATE
2021-05-28 03:56:13,821 DEBUG quic [3cf854ef2dee6084] TLS State.CLIENT_EXPECT_CERTIFICATE_REQUEST_OR_CERTIFICATE -> State.CLIENT_EXPECT_CERTIFICATE_VERIFY
2021-05-28 03:56:13,821 DEBUG quic [3cf854ef2dee6084] TLS State.CLIENT_EXPECT_CERTIFICATE_VERIFY -> State.CLIENT_EXPECT_FINISHED
2021-05-28 03:56:13,822 DEBUG quic [3cf854ef2dee6084] TLS State.CLIENT_EXPECT_FINISHED -> State.CLIENT_POST_HANDSHAKE
2021-05-28 03:56:13,823 ERROR asyncio Exception in callback _SelectorDatagramTransport._read_ready()
handle: <Handle _SelectorDatagramTransport._read_ready()>
Traceback (most recent call last):
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/usr/lib/python3.6/asyncio/selector_events.py", line 1068, in _read_ready
    self._protocol.datagram_received(data, addr)
  File "/home/chris/TVOQE_UPGRADE_27/quic/aioquic/src/aioquic/asyncio/protocol.py", line 135, in datagram_received
    self._quic.receive_datagram(cast(bytes, data), addr, now=self._loop.time())
  File "/home/chris/TVOQE_UPGRADE_27/quic/aioquic/src/aioquic/quic/connection.py", line 923, in receive_datagram
    context, plain_payload
  File "/home/chris/TVOQE_UPGRADE_27/quic/aioquic/src/aioquic/quic/connection.py", line 2058, in _payload_received
    frame_handler(context, frame_type, buf)
  File "/home/chris/TVOQE_UPGRADE_27/quic/aioquic/src/aioquic/quic/connection.py", line 1420, in _handle_crypto_frame
    self._parse_transport_parameters(ext_data)
  File "/home/chris/TVOQE_UPGRADE_27/quic/aioquic/src/aioquic/quic/connection.py", line 2109, in _parse_transport_parameters
    quic_transport_parameters = pull_quic_transport_parameters(Buffer(data=data))
  File "/home/chris/TVOQE_UPGRADE_27/quic/aioquic/src/aioquic/quic/packet.py", line 362, in pull_quic_transport_parameters
    assert buf.tell() == param_start + param_len
AssertionError
2021-05-28 03:56:13,999 DEBUG quic [3cf854ef2dee6084] Loss detection triggered
2021-05-28 03:56:14,000 DEBUG quic [3cf854ef2dee6084] Discarding epoch Epoch.INITIAL
2021-05-28 03:56:14,161 DEBUG quic [3cf854ef2dee6084] Discarding epoch Epoch.HANDSHAKE
2021-05-28 03:56:15,000 DEBUG quic [3cf854ef2dee6084] Loss detection triggered
2021-05-28 03:56:15,000 DEBUG quic [3cf854ef2dee6084] Sending PING (probe) in packet 4
2021-05-28 03:56:15,633 DEBUG quic [3cf854ef2dee6084] Received PING (probe) response
2021-05-28 03:56:16,924 DEBUG quic [3cf854ef2dee6084] Loss detection triggered
2021-05-28 03:56:16,924 DEBUG quic [3cf854ef2dee6084] Sending PING (probe) in packet 5
2021-05-28 03:56:17,562 DEBUG quic [3cf854ef2dee6084] Received PING (probe) response
2021-05-28 03:56:18,537 DEBUG quic [3cf854ef2dee6084] Loss detection triggered
2021-05-28 03:56:18,537 DEBUG quic [3cf854ef2dee6084] Sending PING (probe) in packet 6
2021-05-28 03:56:19,481 DEBUG quic [3cf854ef2dee6084] Received PING (probe) response
2021-05-28 03:56:20,261 DEBUG quic [3cf854ef2dee6084] Loss detection triggered
2021-05-28 03:56:20,261 DEBUG quic [3cf854ef2dee6084] Sending PING (probe) in packet 7
2021-05-28 03:56:22,066 DEBUG quic [3cf854ef2dee6084] Received PING (probe) response
2021-05-28 03:56:23,042 DEBUG quic [3cf854ef2dee6084] Loss detection triggered
2021-05-28 03:56:23,042 DEBUG quic [3cf854ef2dee6084] Sending PING (probe) in packet 8
2021-05-28 03:56:24,547 DEBUG quic [3cf854ef2dee6084] Received PING (probe) response
2021-05-28 03:56:26,296 DEBUG quic [3cf854ef2dee6084] Loss detection triggered
2021-05-28 03:56:26,296 DEBUG quic [3cf854ef2dee6084] Sending PING (probe) in packet 9
